### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by katykennington**
## Changes
- Added initial authentication wrapper
- Added initial api

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,10 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+
+# Pycharm files
+.idea/
```
Comment (JWKennington) [inline .gitignore:108]: this is good for pycharm developers

File: tests/test_auth.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import tweepy
+import unittest
+from twentiment import auth
+
+
+class AuthTest(unittest.TestCase):
```
Comment (JWKennington) [inline tests/test_auth.py:6]: nice tests!

File: twentiment/api.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"""
+api
```
Comment (JWKennington) [inline twentiment/api.py]: Better docstring

File: twentiment/auth.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    from twentiment import secrets
+except ImportError:
+    secrets = None
+
+
+class AuthenticationError(ValueError):
+    pass
+
+
+def authenticate(consumer_key=None, consumer_secret=None, access_token=None, access_secret=None) -> tweepy.OAuthHandler:
```
Comment (JWKennington) [inline twentiment/auth.py:17]: better docstring

**Review by JWKennington — CHANGES_REQUESTED** [review]
This looks good! A few minor documentation requests

File: .gitignore (hunk 1 above)
Comment (katykennington) [reply .gitignore:108]: agreed, we both use pycharm

File: twentiment/api.py (hunk 3 above)
Comment (katykennington) [reply twentiment/api.py]: Improved docstring

File: twentiment/auth.py (hunk 4 above)
Comment (katykennington) [reply twentiment/auth.py:17]: Done

**Review by JWKennington — APPROVED** [review]
Looks good!

**Pull request merged**